### ISSUE TITLE
Train.set_next_split_steering_decision(SteeringDecision.STEER) throws exception

### ISSUE THREAD

**Issue by chronakis** [intelino-code/intelino-trainlib-py#2]
As it says, trying to set the next steering decision to just SteeringDecision.STEER will throw an exception. It works only when explicitly setting it to LEFT or RIGHT.

It would be great for this to work given that it can be a bit confusing.

Mac OS 11.6.1, python 3.9.9, intelino library cloned from sources (version 1.0.0).

**Comment by petrusek** [comment]
Yes, it should throw something like `TrainCommandError` with a message `Parameter SteeringDecision.STEER is not supported!`.

For now it accepts only `LEFT`, `RIGHT` and `STRAIGHT`.

It is planned for some future FW release to support also `STEER`, but I see now that we didn't document it properly. I will either add a note about this or maybe remove the `STEER` key from the enum until we are ready.

Thank you for the feedback. I will bump the priority of this planned feature.

**Off topic:** We have a similar feature in our app under the "Snap editor -> Splits", for "cyan - red/blue - magenta" snaps. It is a "pre-programmed decision buffer" to solve some puzzles. Some of our challenges here https://lab.intelino.com/activities use it.

**Comment by chronakis** [comment]
Thank you for clarifying it. I will be waiting for the future release.

I have completely missed the challenges and the whole labs site! Thanks to the little off-topic note, I can now get back with more activities for my son :)
